core/server/blog/app.js

```diff
@@ -48,6 +47,12 @@ module.exports = function setupBlogApp() {
     blogApp.use(serveSharedFile('sitemap.xsl', 'text/xsl', utils.ONE_DAY_S));
     // Serve robots.txt if not found in theme
     blogApp.use(serveSharedFile('robots.txt', 'text/plain', utils.ONE_HOUR_S));
+
+    // Serve stylesheets for default templates
+    blogApp.use(serveSharedFile('normalizer.css', 'text/css', utils.ONE_HOUR_S));
+    blogApp.use(serveSharedFile('error.css', 'text/css', utils.ONE_HOUR_S));
+    blogApp.use(serveSharedFile('subscribers.css', 'text/css', utils.ONE_HOUR_S));
```

Comment:
Instead of using three different css files, you could simplify it and just serve all the styles from one file instead.